### DISCUSSION TITLE
Speed cameras, fuel stations...

### DISCUSSION THREAD

**Discussion by Cinemannix** [devemux86/cruiser#297] [Q&A] [answered]
Hi, i have two Phones, one AS GPS for the motocycle, and my Mainphone. On the motocycle everything is fine. But in the Mainphone, when I use Cruiser vor navigation, there will not shown any cams or fuelstation on the Phone. Config is the same. The only different is, on mainphone BRouter calculates the whole route, in the motocycle I use kurviger..m

BR

Cine

**Comment by devemux86** [comment, marked as answer]
Hi @Cinemannix,

Have you done the following 2 steps:
- Download / activate the POI for your region (image 1)
Menu "Downloads | POI" (they are automatically downloaded with the maps)
- Enable the display / voice alerts for the POI (image 2)
"Settings | Navigation | Display | Nearby points of interest"
"Settings | Voice guidance | Announcement | Nearby points of interest"

<img src="https://github.com/devemux86/cruiser/assets/3484020/9cc4482a-158f-416c-bd28-ce4f85bbc3a2" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/d61c4250-ac76-49a5-aecf-e4b3857008ea" width="50%"/>

**Comment by Cinemannix** [comment]
Hi, 

looks, there is no german poi... Map is already. Normaly it update both...map and poi... I try to downlad the missing poi, but for the moment IT seems to be offline

**Reply by devemux86** [reply]
The download server appears to be currently offline.

**Reply by devemux86** [reply]
@Cinemannix the download server is online again.

**Reply by Cinemannix** [reply]
Yeah, now IT works, was my fault 🫣

**Reply by Cinemannix** [reply]
Another question, is there a topic about tunnels?

**Reply by devemux86** [reply]
> Another question, is there a topic about tunnels?

Do you mean that Cruiser displays with distance and announces when tunnels are ahead of us on the route?
Like the fuel stations, speed cameras, etc.

**Reply by Cinemannix** [reply]
No, I had some trouble in Italy and Austria inside Gallery and Tunnels. There is no GPS an so the Pointer for position hangs. Sometime there is a turn right inside the tunnel or directly after it. 
I'm not shure, but I think in older kurviger 2 it works (sometimes). Is it not possible to use the averagespeed to follow the track? On map is the Info for Tunnel, löst the GPS, use the average of the last 2 km...

BR

Cine

**Reply by 0709wiwiwi** [reply]
Whether it can be done in Cruiser I don't know. But at least the proposal seems well thought out to me.

**Reply by devemux86** [reply]
@Cinemannix

K1 and K2 use the Cruiser libraries, so they have similar (but older) navigation.

I plan to check what is possible to do inside tunnels without GPS. :slightly_smiling_face:

**Reply by Cinemannix** [reply]
This will be great. This is the last (small) point for me. All the rest works perfect.
Other things like day/night mapmode are in process, or not possible to fix like the GPS delay (50-100 Meters)

**Reply by Cinemannix** [reply]
Here again is my layman's introduction to the tunnel topic:

Entering the tunnel -> GPS gone
Map says tunnel, app switches to simulation mode and takes the average speed or the permitted speed if available in the map material. 

BR

Cine

**Reply by airald** [reply]
Something similar did tomtom app I ran on an HP ipaq many years ago before smartphones appear. When you enter a tunnel, continues following at the same speed. If there were no signal when tomtom supposed you must exit the tunnel, it waits at the tunnel exit to achieve GPS signal again.

**Reply by devemux86** [reply]
Something similar for tunnels is already in my implementation list. :slightly_smiling_face:

**Reply by ramondevic** [reply]
And it would also be good if the tunnel curved to follow the route.
![Screenshot_20230905-175026](https://github.com/devemux86/cruiser/assets/122691401/4d5efb05-bffd-4bd9-b8f5-3b6c20b7b024)
